Config/config.php

```diff
@@ -3,23 +3,50 @@
 return [
     'name'        => 'Housekeeping by Leuchtfeuer',
     'description' => 'Database Cleanup Command to delete lead_event_log table entries, campaign_lead_event_log table entries, email_stats table entries where the referenced email entry is currently not published and email_stats_devices table entries.',
-    'version'     => '1.4.0',
+    'version'     => '1.4.3',
     'author'      => 'Leuchtfeuer Digital Marketing GmbH',
     'services'    => [
+        'integrations' => [
+            'mautic.integration.housekeepingleuchtfeuer' => [
+                'class' => \MauticPlugin\LeuchtfeuerHousekeepingBundle\Integration\HousekeepingLeuchtfeuerIntegration::class,
+                'tags'  => [
```

Comment:
To keep a streamlined naming convention, the class-name should be `LeuchtfeuerHousekeepingIntegration`

Reply:
I tried it already before with `LeuchtfeuerHousekeepingIntegration` (and once again after your comment), but although I changed 'INTEGRATION_NAME' to 'leuchtfeuerhousekeeping' and all config entries to 'leuchtfeuerhousekeeping' the `Enable/Disable` switch does not work. Thatswhy I integrated the solution in my first commit with the class-name 'HouskeepingLeuchtfeuerIntegration' (the Enable/Disbale Switch is shown and after injecting at the right place also working). 

So I finally kept the class-name 'HouskeepingLeuchtfeuerIntegration' with my current commit and did not apply your suggestion (sorry).

Reply:
OK thats strange